data/api.go

```diff
@@ -102,10 +102,9 @@ func (a *API) Get501(res http.ResponseWriter, req *http.Request) {
 // @Summary Get the api status
 // @Description Get the api status
 // @ID tide-whisperer-api-getstatus
-// @Accept json
 // @Produce json
-// @Success 200
-// @Failure 500 {string} string "error description"
+// @Success 200 {object} status.ApiStatus
+// @Failure 500 {object} status.ApiStatus
```

Comment:
The Status struct is not renderered in the final swagger definition
```yaml
 status.ApiStatus:
    properties:
      status:
        type: Status
      version:
        type: string
    type: object
```

Reply:
indeed it is not. I did not find a way to make it. I think this is a drawback of the tool with deep models.

Reply:
using _--parseDependency_ in the _init_ command is making the trick